profiler/src/main/java/com/splunk/opentelemetry/profiler/JfrRecorder.java

```diff
@@ -66,7 +68,10 @@ Recording newRecording() {
 
   public void flushSnapshot() {
     try (Recording snap = jfr.takeSnapshot()) {
-      Path file = Paths.get(Instant.now().toString() + ".jfr");
+      String prefix =
+          DateTimeFormatter.ISO_DATE_TIME.format(
+              LocalDateTime.now().truncatedTo(ChronoUnit.SECONDS));
+      Path file = Paths.get(prefix + ".jfr");
       Path path = outputDir.resolve(file);
       logger.debug("Flushing a JFR snapshot: {}", path);
       snap.dump(path);
```

Comment:
do we worry about the filesystem filling up ? Or not even being writable?

Reply:
Not yet, no.  There will be a component soon that deletes the files after they are read and there will also be a cleanup shutdown hook.

Reply:
If the filesystem isn't writable, this won't work.  I suppose a future effort would be to use the streaming APIs for later versions of java if available.